_posts/2020-07-15-from-engineering-to-product-management-and-back-again.md

```diff
@@ -0,0 +1,261 @@
+I realized I could achive that balance better as an empowered engineer rather than as a product manager.
+
+After a lot of thought, I decided that my longer term career direction made more sense as an engineering leader
+rather than as a product leader. Fortunately Artsy's leadership agreed to let me transition back into engineering,
+and at the beginning of June I was re-minted as a software engineer.
+
+## Onboarding the second time
+
+At this point, I'm six weeks back into engineering and my experience feels very different than before. I've spent
+most of my time ramping up and onboarding for the second time. Relearning our stack has been both a humbling and
+rewarding experience. I've forgotten a lot of the details of the frameworks we use, but this time instead of racing
+to get through tickets I've decided to really slow down and take the time to read the docs and do tutorials to make
+sure I fully understand what's going on. While it comes at the expense of speed in the short-term it'll make me a
+stronger engineer down the road, and I know that tradeoff is worth it. I also can't speak highly enough of my
+fellow engineers, who have been incredibly supportful while I learn.
+
+## Provide value, not code
+
+I've also found my perspective is very different than before. While previously I would have seen my role as writing
+great code, now I see my role as leveraging technology to provide value to users. What's the difference? In the
+former, I'm doing a _task_ while in the latter I'm driving _outcomes_. It feels much more empowering as a mental
+model for what I should do as an engineer. It's also helped me shed any hint of perfectionism I had before since
+writing "perfect" code or building the "perfect" system usually isn't relevant to providing value to our users.
+
+## Developer experience is a product
+
+Working as a product manager really taught me the value of engineering, which is frequently the bottleneck in
+delivering value to your users. Creating the right strategy and vision for a product is critical work, but none of
+that work matters if you can't quickly get it to market. The faster you can move the more quickly you can create
+the ideal product your users want, which in turn dictates how successful your product will be. Gains in engineering
+efficiency snowball into massive benefits for users over time.
+
+That's led me to start seeing our developer experience as part of our product. It's much more than just our
+toolchain -- it's the end-to-end cycle of working with a designer and product manager to understand product
+requirements to building the feature to shipping it and monitoring in production. What does the ideal process look
+like? Where are our bottlenecks? What investments will result in the best outcome for our users? These are
+questions I'm asking myself as I ramp up, and I'm excited to dig into this more.
```

Comment:
This is a really exciting paragraph! MX has been focusing a lot on developer experience since we created the team in September, and this paragraph has given me a new perspective on its importance 👍